### PR TITLE
chore: documentation of task output

### PR DESCRIPTION
**Pull request by nbrouand**

### CODE REVIEW

File: dsl/src/main/kotlin/com/chutneytesting/kotlin/dsl/ChutneyStepImplExtensions.kt [hunk 1]
```diff
@@ -2,18 +2,27 @@ package com.chutneytesting.kotlin.dsl
...
+ * Always fail
+ */
 fun ChutneyStepBuilder.FailTask() {
     Implementation {
         type = "fail"
     }
 }
 
+/**
+ * Log variables of the execution context
```
Comment (bessonm) [inline dsl/src/main/kotlin/com/chutneytesting/kotlin/dsl/ChutneyStepImplExtensions.kt:24]: Log values in the execution context ?

File: dsl/src/main/kotlin/com/chutneytesting/kotlin/dsl/ChutneyStepImplExtensions.kt [hunk 2]
```diff
@@ -552,7 +700,14 @@ fun ChutneyStepBuilder.HttpsServerStartTask(
         validations = validations
     )
 }
+// fun ChutneyStepBuilder.HttpsServerStopTask auto registered by start task
```
Comment (bessonm) [inline dsl/src/main/kotlin/com/chutneytesting/kotlin/dsl/ChutneyStepImplExtensions.kt:703]: why not add this comment to the previous block ?

File: dsl/src/main/kotlin/com/chutneytesting/kotlin/dsl/ChutneyStepImplExtensions.kt [hunk 3]
```diff
@@ -1145,6 +1417,9 @@ fun ChutneyStepBuilder.SeleniumRemoteDriverInitTask(
     if (strategy != null) this.strategy = strategy
 }
 
+/**
+ * Selenium resize brower
```
Comment (bessonm) [inline dsl/src/main/kotlin/com/chutneytesting/kotlin/dsl/ChutneyStepImplExtensions.kt:1421]: browser typo

File: dsl/src/main/kotlin/com/chutneytesting/kotlin/dsl/ChutneyStepImplExtensions.kt [hunk 4]
```diff
@@ -23,13 +32,19 @@ fun ChutneyStepBuilder.DebugTask(filters: List<String> = listOf()) {
     }
 }
 
+/**
+ * Wait "5 min", "300 sec" or "500 ms" time
```
Comment (boddissattva) [inline dsl/src/main/kotlin/com/chutneytesting/kotlin/dsl/ChutneyStepImplExtensions.kt:36]: Add example with until xxxx form maybe

File: dsl/src/main/kotlin/com/chutneytesting/kotlin/dsl/ChutneyStepImplExtensions.kt [hunk 5]
```diff
@@ -38,6 +53,9 @@ fun ChutneyStepBuilder.ContextPutTask(entries: Map<String, Any>, outs: Map<Strin
     }
 }
 
+/**
+ * Add a final task, execute at the end of the execution
```
Comment (boddissattva) [inline dsl/src/main/kotlin/com/chutneytesting/kotlin/dsl/ChutneyStepImplExtensions.kt:57]: ... in a teardown final step

File: dsl/src/main/kotlin/com/chutneytesting/kotlin/dsl/ChutneyStepImplExtensions.kt [hunk 6]
```diff
@@ -141,6 +175,11 @@ fun ChutneyStepBuilder.AmqpUnbindQueueTask(
     if (strategy != null) this.strategy = strategy
 }
 
+/**
+ * On an amqp target, publish a message
+ * @output payload : the payload sent  (String)
```
Comment (boddissattva) [inline dsl/src/main/kotlin/com/chutneytesting/kotlin/dsl/ChutneyStepImplExtensions.kt:180]: Remove @output.

File: dsl/src/main/kotlin/com/chutneytesting/kotlin/dsl/ChutneyStepImplExtensions.kt [hunk 7]
```diff
@@ -1279,6 +1577,9 @@ fun ChutneyStepBuilder.XsdValidationTask(
     )
 }
 
+/**
+ * Compare two String on multiple mode (equals, not-equals, contains, not-contains, greater-than, less-than
```
Comment (boddissattva) [inline dsl/src/main/kotlin/com/chutneytesting/kotlin/dsl/ChutneyStepImplExtensions.kt:1581]: close parenthesis.
For greater and less comparator, numerics are expected.

**Review by boddissattva — CHANGES_REQUESTED** [review]

**Pull request merged**